editor/import/resource_importer_texture.cpp

```diff
@@ -540,6 +557,13 @@ Error ResourceImporterTexture::import(const String &p_source_file, const String
 			//convert to ldr if this can't be stored hdr
 			image->convert(Image::FORMAT_RGBA8);
 		}
+		if (pack_channels == 1) { // Discard A channel.
```

Comment:
Did you intend this to be an `else if` I provided two options because I wasn't sure but `} if` would not be valid syntax

Reply:
It should be `else if`, don't want two reconversions

Reply:
I wasn't familiar with the details of the code that's why I offered two versions and let you decide because you are familiar with it